packages/compass/src/app/components/home.spec.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { type ComponentProps } from 'react';
 import {
   cleanup,
+  fireEvent,
```

Comment:
We usually use @testing-library/user-events instead, not fireEvent directly (as suggested by testing-library docs)